scripts.js

```diff
@@ -11,6 +11,6 @@ console.log(textAfter);
 
 console.log(textAfter.length/2);
 
-var textSliced = text.slice(0, 73);
+var textSliced = text.slice(0, 72);
```

Comment:
i want you to assign your calculation to new variable and than use this variable in this line :) when you will have an real function you cannot add by your own values - everything should be calculated in your code ;)